### DISCUSSION TITLE
怎么修改页面的名字啊？

### DISCUSSION THREAD

**Discussion by elarbor** [JackySoft/marsview#44] [Q&A]
![image](https://github.com/user-attachments/assets/a1dc5ad2-2410-454b-95c3-d5ebe77f354f)

![image](https://github.com/user-attachments/assets/d1195720-abc2-4a8e-8c7a-eda69df05f06)

**Comment by JackySoft** [comment]
![image](https://github.com/user-attachments/assets/710995d7-76b0-42cc-8385-8442244e1cb2)
这里可以修改。
